.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,80 @@
+name: Bug Report
+about: Report an issue related to the Home Assistant Supervisor.
+labels: bug
+title: ""
+issue_body: true
+inputs:
+  - type: description
+    attributes:
+      value: |
+        This issue form is for reporting bugs with **supported** setups only!
+
+        If you have a feature or enhancement request, please use the [feature request][fr] section of our [Community Forum][fr].
+        [fr]: https://community.home-assistant.io/c/feature-requests
+  - type: input
+    attributes:
+      label: What is the version of the Supervisor used?
+      required: true
+      placeholder: supervisor-
+      description: >
+        Can be found in the Supervisor panel -> System tab. Starts with
+        `supervisor-....`.
+  - type: dropdown
+    attributes:
+      label: What type of installation are you running?
+      required: true
+      description: >
+        If you don't know, you can find it in: Configuration panel -> Info.
+      choices:
+        - Home Assistant OS
+        - Home Assistant Supervised
+  - type: dropdown
+    attributes:
+      label: Which operating system are you running on?
+      required: true
+      choices:
+        - Home Assistant Operating System (HassOS)
```

Comment:
```suggestion
        - Home Assistant Operating System (HAOS)
```

Reply:
HassOS is still displayed. I did not try to shorten the installation method, instead, it was a reference to the operating system itself (which still is displayed as HassOS)

Reply:
Where is it displayed like that?

Reply:
Not that many places, I agree. Let's remove the abbreviation completely then. Agree?

Reply:
That works too :+1: